
class ExceptionCustom(Exception):
    pass

class EmptyError(Exception):
    pass